django-data/image/submissions/views.py

```diff
@@ -140,15 +141,34 @@ def get_context_data(self, **kwargs):
         context = super(DetailSubmissionView, self).get_context_data(**kwargs)
 
         # add submission report to context
-        validation_summary = ValidationSummary(self.submission)
+        validation_summary = VS(self.submission)
```

Comment:
This need to use the `models.ValidationSummary`  not more `helpers.ValidationSummary`. Please, inspect the Django Debug Toolbar (you should reach your local installation using http://localhost:26080/image/submissions/1/, for example) click on `DjDT` on the right side and then click over `SQL` tab: you will see all the queries performed while visualizing this page. Could we load data from ValidationSummary model instead?

Reply:
Yes, sure. Done!